site/contents/write_ups/2021/15-11-21_writeup.md

```diff
@@ -0,0 +1,85 @@
+# Data Ethics Club: [A Question of Trust](https://www.bbc.co.uk/programmes/p00ghvd8)
+
+```{admonition} What's this? 
+This is summary of Wednesday 15th December's Data Ethics Club discussion, where we spoke and wrote about the lecture [A Question of Trust](https://www.bbc.co.uk/programmes/p00ghvd8) by Onora O'Neill.
+
+The summary was written by Huw Day, who tried to synthesise everyone's contributions to this document and the discussion. "We" = "someone at Data Ethics Club". 
+Nina Di Cara and Natalie Thurlby helped with a final edit.
+```
+
+## Do you think that the public trust data science as a profession? What makes a data scientist trustworthy?
+
+It is important to note immediately that this lecture was aired in 2002, a time where social media was much less prominent and public trust in science was typically much higher. Whilst we may not have done enough to keep this trust, misinformation has also had a big impact. At times throughout history, science has developed from something not very trustworthy and has perhaps had already peaked.
+
+Some people have the viewpoint that anyone who does not trust scientists/doctors/etc. is a conspiracy theorist or simply not worth talking to. This can be extremely polarising. It is also problematic to assume that everyone from a particular group or progression has the same views.
+
+We were unable to come to a consesus as to whether this phenomenom is getting better or worse. Intellectual superiority in universities reinforces this mindset, creating a bubble of knowledge where you become ignorance of other people's (or indeed your own) ignorance. It is important that we can admit that we are wrong but the culture in academia is not very conducive when it comes to encouraging this. There is very public evidence of people being humiliated by colleagues for making relatively minor mistakes. Issues of irreproducibility in scientific results can also undermind trust.
+
+As the current Covid-19 pandemic has bought scientific advice to the forefront of how we live our lives. Perhaps as a result of this, the public trust statisticians less, same as the weatherman. There is a difference between what people describe as a data scientist versus a data analyst. 
```

Comment:
I think this was the implication here? Just trying to make the point a bit clearer
```suggestion
The current Covid-19 pandemic has bought scientific advice to the forefront of how we live our lives. Perhaps as a result of this, the public trust statisticians less, same as the weatherman who makes predictions that frequently turn out to be inaccurate. There is a difference between what people describe as a data scientist versus a data analyst. 
```